File: service/src/main/java/com/coopra/service/interfaces/TracksInterface.java
package com.coopra.service.interfaces;

import com.coopra.data.Track;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface TracksInterface {
    @GET("tracks")
    Call<List<Track>> getFeedTracks(@Query("client_id") String clientId);
}

File: service/src/main/java/com/coopra/service/interfaces/TracksInterface.kt
package com.coopra.service.interfaces

import com.coopra.data.Origin
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface TracksInterface {
    @GET("tracks")
    fun getRandomTracks(@Query("client_id") clientId: String): Call<List<Origin>>
}